tensorflow/training/docker/2.8/py3/Dockerfile.cpu

```diff
@@ -140,29 +133,144 @@ RUN ${PIP} install --no-cache-dir -U \
     "Pillow>=9.0.0<9.1.0" \
     python-dateutil==2.8.1 \
     "pyYAML>=5.4,<5.5" \
-    requests==2.24.0 \
+    requests==2.27.1 \
     "awscli<2" \
     mpi4py==3.0.3 \
     # Let's install TensorFlow separately in the end to avoid
     # the library version to be overwritten
  && ${PIP} install --no-cache-dir -U \
-    ${TF_URL} \
-    tensorflow-io==0.24.0 \
     h5py==3.1.0 \
     "absl-py>=0.9,<0.11" \
     "opencv-python>=4.3,<5" \
     "werkzeug>=2.0.2,<3" \
     psutil==5.7.2 \
    "urllib3>1.26.0<1.27.0"
 
+ADD https://raw.githubusercontent.com/aws/deep-learning-containers/master/src/deep_learning_container.py /usr/local/bin/deep_learning_container.py
+
+RUN chmod +x /usr/local/bin/deep_learning_container.py
+
+RUN curl https://aws-dlc-licenses.s3.amazonaws.com/tensorflow-2.8/license.txt -o /license.txt
```

Comment:
suggestion: Can we make `2.8` as `TF_VERSION` arg so that this line doesn't need to be manually changed for every release?

Reply:
This sounds like a good idea!! @hsl89 , can we do that to prevent changing it manually in the future?

Reply:
That's a good idea. Added it in fe80164

Reply:
done